new-lamassu-admin/src/pages/Customers/CustomerData.js

```diff
@@ -120,44 +131,56 @@ const CustomerData = ({
     }),
     frontCamera: Yup.object().shape({
       frontCamera: Yup.mixed().required()
-    })
+    }),
+    smsData: Yup.object()
+      .shape({
+        phoneNumber: Yup.string().required()
+      })
+      .required()
```

Comment:
I believe we need this .required() to use concat() further down, otherwise, I think Yup's type definition will add undefined as an intersection type since we didn't specify the given object would always be not-null.